shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Deleg.hs

```diff
@@ -102,7 +102,7 @@ data DelegPredicateFailure era
   | StakeDelegationImpossibleDELEG
       !(Credential 'Staking era) -- Credential that is not registered
   | WrongCertificateTypeDELEG -- The DCertPool constructor should not be used by this transition
-  | GenesisKeyNotInpMappingDELEG
+  | GenesisKeyNotInMappingDELEG
```

Comment:
@mrBliss , will it cause you any trouble if we change the name of this constructor? My thought was that while changing the serialization for the predicate failures is problematic, this should be trivial. it's an unfortunate typo.

Reply:
(Oops, missed this question). This is indeed no problem, as it doesn't affect the serialisation format.